templates/platform_landing_zone/modules/management_groups/main.tf

```diff
@@ -1,7 +1,7 @@
 module "management_groups" {
   source                                                        = "Azure/avm-ptn-alz/azurerm"
   version                                                       = "0.12.2"
-  architecture_name                                             = try(var.management_group_settings.architecture_name, "alz")
+  architecture_name                                             = try(var.management_group_settings.architecture_name, "alz_custom")
```

Comment:
Per the other comment, there was a reason for overriding the `alz` library, but I don't recall what it was...